sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/README.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft Azure Data SDK for Python
```

Comment:
There shouldn't be a readme or any setup files in this directory.
The setup for the namespace package should be in a separate directory:
`sdk/digitaltwins/azure-digitaltwins-nspkg`

Reply:
Fixed